src/api/journeys.ts

```diff
@@ -1,24 +1,47 @@
-import { QueryState } from 'react-query/types/core/query';
-import { UseQueryResult } from 'react-query';
-
+/* eslint-disable @typescript-eslint/explicit-module-boundary-types */
+/* Reason for above: the output types of these resources are cast
```

Comment:
We have this eslint-disable in every api file. I don't think we really need this line. And I don't think this is why it's needed. I think it's needed because it's annoying to type everything in the return object and typescript is smart enough to infer the return type without explicitly setting it.

Reply:
Done